iothub/service/src/ServiceClient.cs

```diff
@@ -76,7 +76,7 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         public static ServiceClient Create(
             string hostName,
             TokenCredential credential,
-            TransportType transportType,
+            TransportType transportType = TransportType.Amqp,
```

Comment:
Following the pattern we have for the connection string constructors where the default is AMQP.